spec/index.bs

```diff
@@ -1324,7 +1330,98 @@ dictionary GPUBufferBinding {
+1. For each {{GPUBindGroupBinding}} |bindingDescriptor| in
+    |descriptor|.{{GPUBindGroupDescriptor/bindings}}:
+    1. Ensure there is exactly one {{GPUBindGroupLayoutBinding}} |layoutBinding| in
+        {{GPUBindGroupLayoutDescriptor/bindings}} of |descriptor|.{{GPUBindGroupDescriptor/layout}}
+        such that |layoutBinding|.{{GPUBindGroupLayoutBinding/binding}} equals to
+        |bindingDescriptor|.{{GPUBindGroupBinding/binding}}.
+    1. If |layoutBinding|.{{GPUBindGroupLayoutBinding/type}} is
+        {{GPUBindingType/"sampler"}}:
+        1. Ensure |bindingDescriptor|.{{GPUBindGroupBinding/resource}} is a
+            valid {{GPUSampler}} object.
+    1. If |layoutBinding|.{{GPUBindGroupLayoutBinding/type}} is
+        {{GPUBindingType/"sampled-texture"}} or {{GPUBindingType/"storage-texture"}}.
+        1. Ensure |bindingDescriptor|.{{GPUBindGroupBinding/resource}} is a
+            valid {{GPUTextureView}} object.
+        1. Ensure [=texture view binding validation=] is not violated.
+    1. If |layoutBinding|.{{GPUBindGroupLayoutBinding/type}} is
+        {{GPUBindingType/"uniform-buffer"}} or {{GPUBindingType/"storage-buffer"}}
+        or {{GPUBindingType/"readonly-storage-buffer"}}.
+        1. Ensure |bindingDescriptor|.{{GPUBindGroupBinding/resource}} is a
+            valid {{GPUBufferBinding}} object.
+        1. Ensure [=buffer binding validation=] is not violated.
+1. Return a new {{GPUBindGroup}} object with:
+    - {{GPUBindGroup/[[layout]]}} = |descriptor|.{{GPUBindGroupDescriptor/layout}}
+    - {{GPUBindGroup/[[bindings]]}} = |descriptor|.{{GPUBindGroupDescriptor/bindings}}
+
+<b>Validation Conditions</b>
+
+<dl dfn-for="createBindGroup(descriptor)">
+If any of the following conditions are violated:
+    1. Generate a {{GPUValidationError}} in the current scope with appropriate error message.
+    1. Create a new invalid {{GPUBindGroup}} and return the result.
+
+<dfn>bind group device validation</dfn>: The {{GPUDevice}} must not be lost.
+
+<dfn>texture view binding validation</dfn>: A compatible texture BindGroupLayout binding must exist
+    in {{GPUBindGroupLayoutDescriptor/bindings}} of |descriptor|.{{GPUBindGroupDescriptor/layout}}.
+    This requires:
+    1. Its |layoutBinding|.{{GPUBindGroupLayoutBinding/textureDimension}} must be the same with
+        the corresponding texture view's {{GPUTextureViewDescriptor/dimension}}.
+    1. Its |layoutBinding|.{{GPUBindGroupLayoutBinding/textureComponentType}} must be compatible
+        with the corresponding texture view's {{GPUTextureViewDescriptor/format}}.
+    1. If |layoutBinding|.{{GPUBindGroupLayoutBinding/multisampled}} is `true`, the corresponding
+        texture's {{GPUTextureDescriptor/sampleCount}} must be greater than 1.
+        Otherwise, if |bindingDescriptor|.{{GPUBindGroupLayoutBinding/multisampled}} is `false`,
+        the corresponding texture's {{GPUTextureDescriptor/sampleCount}} must be 1.
```

Comment:
```suggestion
        |view|'s texture's {{GPUTextureDescriptor/sampleCount}} must be 1.
```

(and so on)

(We'll tighten up these informal `'s` notations later.)

Reply:
Done.